bot/bot.py

```diff
@@ -0,0 +1,30 @@
+import discord
+from discord.ext import commands
+
+from bot.constants import COMMAND_PREFIX
+
+
+class oSTEMBot(commands.Bot):
```

Comment:
Don't have an empty class, just use `command.Bot` directly. Maybe you're intending for this to be customized later, in which case, it's fine.

Nit: class name doesn't conform to CamelCase; consider using `OstemBot` (mixed feelings on this - but I've seen things like `HttpClient` instead of `HTTPClient` so there's convention for just forcing acronyms into CamelCase like that).

Reply:
I couldn't bring myself to do `OstemBot`, I just couldn't. But the empty class is indeed there for later customization.